src/pages/registration/RegistrationFormActions.tsx

```diff
@@ -100,55 +97,61 @@ export const RegistrationFormActions = ({ tabNumber, setTabNumber, validateForm
             </Tooltip>
           </Box>
         )}
-
-        <Box sx={{ gridArea: 'support-button', display: 'flex', justifyContent: 'start' }}>
-          <Button
-            data-testid={dataTestId.registrationWizard.formActions.openSupportButton}
-            onClick={toggleSupportModal}>
-            {t('common.support')}
-          </Button>
-        </Box>
+        <Button
+          sx={{
+            height: '1.4rem',
+            bgcolor: 'white',
+            borderStyle: 'solid',
+            borderRadius: '8px 0px',
+            borderWidth: '2px 5px',
```

Comment:
ser det er to komponenter i figma, hvor bakgrunnen er 22px høy, mens "knappen" er 20px høy, så da blir det 1px ekstra på hver side
```suggestion
            borderWidth: '1px 5px',
```